components/badge/badge.css

```diff
@@ -0,0 +1,41 @@
+.badge {
+    position: relative;
+}
+
+.badge > div {
+    position: absolute;
+    background-color: var(--color-highlight-orange);
+    height: 10px;
+    width: 10px;
```

Comment:
just a suggestion using relative value like rem will be better

Reply:
I meant to change it but it totally slipped my mind. Thanks for pointing it out!